robot/basestation/static/js/elements/feed/navbar.js

```diff
@@ -134,22 +134,22 @@ $(document).ready(() => {
         buttonText.includes('Rover') &&
         $('#toggle-rover-listener-btn')[0].checked == true
       ) {
-        // sendRoverCommand(cmd) // rover commands not yet implemented
+        // sendRoverCommand(command) // rover commands not yet implemented
       } else if (
         buttonText.includes('Arm') &&
         $('#toggle-arm-listener-btn')[0].checked == true
       ) {
-        sendArmCommand(cmd)
+        sendArmCommand(command)
       } else if (buttonText.includes('Science')) {
         // science buttons unknown
-        // sendScienceCommand(cmd) // science commands not yet implemented
+        // sendScienceCommand(command) // science commands not yet implemented
       } else if (buttonText.includes('PDS')) {
         // pds buttons unknown
-        // sendPdsCommand(cmd) // pds commands not yet implemented
+        // sendPdsCommand(command) // pds commands not yet implemented
       }
       // no listener is open, send generic request
       else if (!buttonText.includes('Select Device Channel')) {
-        requestSerialCommand(cmd, function (msgs) {
+        requestSerialCommand(command, function (msgs) {
           console.log(msgs)
```

Comment:
can you please control + f for console.logs in this file and change them to the appriate roslog? (i.e logInfo, logDebug, logWarn, etc.)

Reply:
Fixed